querier/src/cache/mod.rs

```diff
@@ -125,6 +148,16 @@ impl CatalogCache {
 
     /// Processed tombstone cache.
     pub(crate) fn processed_tombstones(&self) -> &ProcessedTombstonesCache {
-        &self.processed_tombstones
+        &self.processed_tombstones_cache
+    }
+
+    /// Parquet file cache.
+    pub(crate) fn parquet_file(&self) -> &ParquetFileCache {
+        &self.parquet_file_cache
+    }
+
+    /// Parquet file cache.
```

Comment:
Copypasta comment :)

```suggestion
    /// Tombstone cache.
```

Reply:
🤦